### DISCUSSION TITLE
How to unselect a yanked file?

### DISCUSSION THREAD

**Discussion by LinArcX** [sxyazi/yazi#592] [1. Q&A]
When we press `y`, maybe we don't want to continue and we want to un-yank the selected items. how to do that?

**Comment by sxyazi** [comment]
You might be interested in https://github.com/sxyazi/yazi/pull/313

I will merge it in the next few days, I still need to wait for [that RFC](https://github.com/sxyazi/yazi/issues/319) (I've marked it as "waiting on op") since they conflict with each other
